react/components/FilterOptionTemplate.js

```diff
@@ -110,25 +110,67 @@ const FilterOptionTemplate = ({
     )
   }, [filters, searchTerm, thresholdForFacetSearch])
 
+  const openTruncated = value => {
+    if (
+      filtersFetchMore &&
+      isLazyFacetsFetchEnabled &&
+      !truncatedFacetsFetched
+    ) {
+      setTruncatedFacetsFetched(true)
+      filtersFetchMore({
+        variables: {
+          from: FACETS_RENDER_THRESHOLD,
+          to: undefined, // to the end of the results
+        },
+        updateQuery: (prevResult, { fetchMoreResult }) => {
+          if (!prevResult || !fetchMoreResult) {
+            return
+          }
+          const prevFacets = prevResult.facets.facets
+          const newFacets = fetchMoreResult.facets.facets
+          const fullFacets = []
+          for (let i = 0; i < prevFacets.length; i++) {
```

Comment:
how many items does `prevFacets` may have? I mean, is it an array of facet groups or an array of facet items?

Reply:
the length right now is always 10 or less, and this component renders a facet group, so every group (color, size, texture, etc) will run this code!

Also, about the `?.`, it is not possible to do that because JS does not support it 😭